src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String password;
+
+    private String email;
+}
```

Comment:
The User class is missing getter and setter methods for its fields. These are necessary for encapsulating the fields and providing access to them from outside the class.